apps/meteor/ee/server/patches/mergeContacts.ts

```diff
@@ -0,0 +1,33 @@
+import type { ILivechatContact, ILivechatContactChannel } from '@rocket.chat/core-typings';
+import { License } from '@rocket.chat/license';
+import { LivechatContacts } from '@rocket.chat/models';
+
+import { ContactMerger } from '../../../app/livechat/server/lib/ContactMerger';
+import { mergeContacts } from '../../../app/livechat/server/lib/Contacts';
+
+export const runMergeContacts = async (_next: any, contactId: string, visitorId: string): Promise<ILivechatContact | null> => {
+	const originalContact = (await LivechatContacts.findOneById(contactId)) as ILivechatContact;
```

Comment:
```suggestion
	const originalContact = await LivechatContacts.findOneById(contactId);
```

no need for casting, since you're validating it to be non null below